sections/futures/PositionCard/PositionCard.tsx

```diff
@@ -89,10 +88,8 @@ const PositionCard: React.FC<PositionCardProps> = () => {
 			: undefined;
 
 	const thisPositionHistory = useMemo(() => {
-		return positionHistory[futuresAccountType].find(
-			({ asset, isOpen }) => isOpen && asset === marketAsset
-		);
-	}, [positionHistory, marketAsset, futuresAccountType]);
+		return positionHistory.find(({ asset, isOpen }) => isOpen && asset === marketAsset);
+	}, [positionHistory, marketAsset]);
```

Comment:
We reuse this logic in a few different components, maybe should just have a `currentPositionHistory` selector that takes care of this logic in state.

Reply:
yea good call